launch_testing_ros/test/examples/wait_for_topic_launch_test.py

```diff
@@ -57,11 +59,16 @@ def test_topics_successful(self, count):
             """All the supplied topics should be read successfully."""
             topic_list = [('chatter_' + str(i), String) for i in range(count)]
             expected_topics = {'chatter_' + str(i) for i in range(count)}
+            message_pattern = re.compile(r'Hello World: \d+')
 
             # Method 1 : Using the magic methods and 'with' keyword
-            with WaitForTopics(topic_list, timeout=2.0) as wait_for_node_object_1:
+            with WaitForTopics(topic_list, timeout=2.0,
+                               messages_received_buffer_length=10) as wait_for_node_object_1:
                 assert wait_for_node_object_1.topics_received() == expected_topics
                 assert wait_for_node_object_1.topics_not_received() == set()
```

Comment:
The one place we are nuanced is on this point "[We prefer hanging indents for continuation lines](https://www.python.org/dev/peps/pep-0008/#indentation)." (again http://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Code-Style-Language-Versions.html#python).

This is a bit of a weird one, but I'd say this is better:

```suggestion
            with WaitForTopics(
                topic_list, timeout=2.0, messages_received_buffer_length=10
            ) as wait_for_node_object_1:
                assert wait_for_node_object_1.topics_received() == expected_topics
                assert wait_for_node_object_1.topics_not_received() == set()
```

That's odd at first, but imagine this is a function prototype instead, replacing `with` with `def` and `) as wait_for_node_object_1:` with `):`, and I think it makes more sense.